src/main/java/bio/terra/service/dataset/BigQueryPartitionConfigV1.java

```diff
@@ -30,6 +30,8 @@ private BigQueryPartitionConfigV1(Mode mode, String columnName, Long intMin, Lon
         this.intInterval = intInterval;
     }
 
+    public BigQueryPartitionConfigV1() {}
```

Comment:
@RoriCremer ran into this before and I just ran into it again -- Jackson gets upset that this doesn't have a default constructor. @danxmoran any thoughts on this? I assume it probably has something to do with a different version of Jackson ending up on my classpath than what I had before.. this happened immediately after I cleared my cache and re-synced my dependencies.

Reply:
Am unsure, though I remember @ddietterich also hitting an issue while testing my PR that mysteriously disappeared

Reply:
Jackson either needs a creator constructor that fills in the properties or it needs a default constructor and then uses the accessors.

In this particular case, the `version` field is not settable by the constructor so Jackson gets angry.

I don't know why it works sometimes and not others.

Reply:
It only ever works on mine when I add that default constructor as well